cmd/fyne/internal/mobile/gendex/gendex.go

```diff
@@ -14,7 +14,7 @@
 //
 // Requires ANDROID_HOME be set to the path of the Android SDK, and
 // javac must be on the PATH.
-package mobile
+package main
```

Comment:
Have to move into different directory and to `main` package to use with go generate (possibly because of go modules?)

Reply:
cunning. I was always just changing the package name manually ;)

Reply:
Yeah, but a couple of times later I decided to fix it :)